packages/app/ui/buttons/RadioButton.tsx

```diff
@@ -10,26 +11,25 @@ export const radioButtonStyles = cva(
   [
     "flex items-center justify-center rounded-full px-3 py-2 space-x-1.5",
     "cursor-pointer select-none font-medium text-xs",
-    "active:ring-4",
     "disabled:bg-surface-75 disabled:text-em-disabled disabled:cursor-not-allowed disabled:ring-0",
-    "focus:outline-none focus:ring-4",
+    "focus:outline-none"
```

Comment:
Any particular reason to remove this? I think we need to review this as a whole, from buttons to radio buttons. We need to check with Akash what we can improve and what we should delete :)

Reply:
Removed the rings because of how token chips behave in other apps like Uniswap. When you click them you don't get the ring styles because they're usually simple divs, we're using radio buttons to emulate those divs they use as chips.

If we want those rings to be back on, I don't mind it but I think looks a bit weird and ugly

Reply:
@ElRodrigote I agree with you. Lets remove in another PR and Akash can help find something prettier!

https://linear.app/swaprdev/issue/STK-91/redo-buttons-active-and-focus-borders created task for this.